kratos/python_scripts/eigen_solver_factory.py

```diff
@@ -36,7 +40,11 @@ def ConstructSolver(settings):
         if kratos_utils.CheckIfApplicationsAvailable("EigenSolversApplication"):
             import KratosMultiphysics.EigenSolversApplication as EiSA
             if EiSA.HasFEAST():
-                if settings.Has("symmetric") and settings["symmetric"].GetBool():
+                if settings.Has("symmetric"):
```

Comment:
using the ternary operator would be cleaner IMO

Reply:
done